index.html

```diff
@@ -1,92 +1,106 @@
     </p>
     <blockquote>
       <span class="name">Foobar123:</span>
       <p class="message">
-        Any Java experts around who are willing to commit into looking into my
+        jan sona Java lon ma ni li, wile e pona e nimi Java mi? taso Java ala la jan ale li ken ala ken pana e sona tawa mi?
+        <!--Any Java experts around who are willing to commit into looking into my
         problem, whatever that may turn out to be, even if it's not actually
         related to Java or if someone who doesn't know anything about Java
-        could actually answer my question?
+        could actually answer my question?-->
       </p>
     </blockquote>
     <p>
-      There are plenty of reasons why people who DO have the knowledge would
+      jan pi jo sona li pana e sona ala, tan nasin sona mute. tenpo la, sina toki alasa e sona ete.
+      <!--There are plenty of reasons why people who DO have the knowledge would
       not admit to it. By asking, you're asking for more than what you think
-      you're asking.
+      you're asking.-->
     </p>
     <p>
-      You're asking people to take responsibility. You're questioning people's
+      sina wile e jan ete li lawa sona. sina toki alasa e sona jan ete. sina pake e jan ete li pana e sona kin. tenpo mute la, mi pana e sona tawa jan mute, tan nasin pali li sama sama tawa jan nimi linluwi.
+      <!--You're asking people to take responsibility. You're questioning people's
       confidence in their abilities. You're also unnecessarily walling other
       people out. I often answer questions related to languages or libraries I
       have never used, because the answers are (in a programmer kind of way)
-      common sense.
+      common sense.-->
     </p>
     <p>
-      Alternatively, it can be seen as..
+      anu, ni li ken e sona e ni:
+      <!--Alternatively, it can be seen as..-->
     </p>
     <blockquote>
       <span class="name">Foobar123:</span>
       <p class="message">
-        I have a question about Java but I'm too lazy to actually formalize it
+        mi jo e sona wile Java taso, mi wile ala e toki e ni tan, jan jo sona la lon ala li ma ni.
+        <!--I have a question about Java but I'm too lazy to actually formalize it
         in words unless there's someone on the channel who might be able to
-        answer it
+        answer it-->
       </p>
     </blockquote>
     <p>
-      ..which is just lazy. If you're not willing to do the work to solve your
-      problem, why should we?
+      ni li wile ala pali kin. sina wile ala e pona e nimi linluwi sina tan, jan ete li wile tan seme?
+      <!--..which is just lazy. If you're not willing to do the work to solve your
+      problem, why should we?-->
     </p>
     <p>
-      The solution is not to ask to ask, but just to ask. Someone who is idling
+      nasin li ni: o toki alasa ala tan toki alasa a! sina toki alasa e toki pi pakala la, jan seme li lon e toki linluwi li wile e pana sona.
```

Comment:
`toki pi pakala` is ungrammatical because there is only one word after `pi`. that makes `pi` redundant.

the statement `o toki alasa ala tan toki alasa a!` seems like a calque from english which doesn't really make sense lon toki pona. toki pona la, state what you *really* mean.

```suggestion
      nasin pona li ni: o alasa pona taso! jan ante mute li lukin lili taso e lipu. sina alasa ike la, ona li wile ala pona e sina. taso, sina alasa pona la, alasa sina li ken pona tawa ona. ni la, ona li wile pona e sina.
```